palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/SimplePalisadeService.java

```diff
@@ -179,7 +179,7 @@ private void auditRequestReceivedException(final RegisterDataRequest request, fi
         LOGGER.info("Auditing error with audit service");
         AuditRequest auditException = getAuditException(request)
                 .withException(ex)
-                .withServiceClass(serviceClass);
+                .withServiceName(serviceClass.getSimpleName());
         auditService.audit(auditException);
```

Comment:
I'd prefer to change the argument passed to this method too.
Then each of the get FutureXxx would be more like:
```
        }).exceptionally((Throwable ex) -> {
            auditRequestReceivedException(request, ex, "policy-service");
            throw new RegisterRequestException("Exception from policyService", ex);
        });
```

Reply:
Since this is being sent out of the palisade-service to the audit-service, the general pattern on names is "something-service", this isn't too important internal to a service for eg logging but externally I think this consistency would be good (see SimpleConnectionDetail and resources/application.yaml web.client keys (values vary by environment))

Reply:
Would this be better to have `service.getName()`?

Reply:
Would this be better to have `service.getName()`?

Reply:
Actually, come to think of it, would it be better to add `getName` default method to the `PalisadeService` interface?

Reply:
Turns out that services do not have to implement the `uk.gov.gchq.palisade.service.Service` interface. With this being the case, there is no point in putting type and name methods onto it. The downside of this is that it is up external classes to create the name of the service when, for example, throwing an exception.

We'll stick to "something-service".